apps/server/src/db/schema.ts

```
@@ -201,6 +201,76 @@ export const oauthAccessToken = createTable('oauth_access_token', {
+    .references(() => theme.id, { onDelete: 'cascade' }),
+  createdAt: timestamp('created_at').notNull().defaultNow(),
+  updatedAt: timestamp('updated_at').notNull().defaultNow().$onUpdate(() => new Date()),
+}, (table) => {
+  return [
+    primaryKey({
+      columns: [table.connectionId],
+    }),
+  ];
+});
+
+export const theme = createTable('theme', {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'theme'. Consider to delete it or rename it.

'theme' is defined here:


(lint/suspicious/noRedeclare)